src/docs/get-started/test-drive/_androidstudio.md

```diff
@@ -2,13 +2,17 @@
 
 ## Create the app {#create-app}
 
- 1. Select **File > New Flutter Project**.
- 1. Select **Flutter application** as the project type, and press **Next**.
- 1. Make sure that the **Flutter SDK Path** text field specifies the
-    location of the SDK. Install the SDK if you haven't yet done so.
- 1. Enter a project name (for example, `myapp`), and press **Next**.
+ 1. Open the IDE and select **Start a new Flutter project**.
+ 1. Select **Flutter Application** as the project type. Then click **Next**.
+ 1. Verify the Flutter SDK path specifies the SDK’s location (select **Install SDK...** if the text field is blank).
+ 1. Enter a project name (e.g., `myapp`). Then click **Next**.
  1. Click **Finish**.
- 1. Wait for Android Studio to install the SDK, and create the project.
+ 1. Wait for Android Studio to install the SDK and create the project.
+
+note: >-
```

Comment:
1. Break line length into 80 chars or less, preferably on a phrase.
2. We do notes this way:

{{site.alert.note}}
  When creating a new Flutter app, ....
{{site.alert.end}}